core/auth/src/main/java/software/amazon/awssdk/auth/signer/params/AwsS3V4SignerParams.java

```diff
@@ -87,7 +87,7 @@ public static Builder builder() {
     }
 
     private static final class BuilderImpl extends Aws4SignerParams.BuilderImpl<Builder> implements Builder {
-        static final boolean DEFAULT_CHUNKED_ENCODING_ENABLED = true;
+        static final boolean DEFAULT_CHUNKED_ENCODING_ENABLED = false;
```

Comment:
See my comment in the description. Only `PutObject` and `UploadPart` support chunked signing. If you try to do it with an operation like `PuObjecTtagging` S3 returns an exception. We still default to `true` for `PutObject` and `UploadPart` via the interceptor.

Reply:
It also makes little sense for the other operations IMO since the reason you'd want chunked encoding is to avoid reading large streams to calculate the content SHA.